src/config/envs/production.ts

```diff
@@ -0,0 +1,20 @@
+import { AppConfig } from '../app-config.interface';
+import { defineConfig } from '../define-config';
+import { AppEnv } from '../env.enum';
+
+// anything that is not a secret can go in here, get the rest from env
+
+export function getProductionConfig(): AppConfig {
+    return defineConfig({
+        env: AppEnv.PROD,
+        frontend: {
+            url: 'https://www.example.com'
+        },
+        backend: {
+            url: 'https://api.example.com'
+        },
+        auth: {
+            providers: ['google', 'onelogin']
+        }
+    });
+}
```

Comment:
I don't think it should be hard coded here because we'll have multiple "production" type environments as we'll have Marvells staging environment which should behave as if it were production, staging on the welsh gov side and finally the full production environment on the welsh gov side.

Reply:
It's easy with this setup to have multiple staging and prod envs if we need them, they just need adding to the `env.enum` (`marv-staging`, `wg-prod` or whatever) and their own config file.

The benefit of hardcoding non-secret stuff is that you don't have to manage or go hunting for the env vars to find out what the value is.

But happy to remove the hardcoded stuff from this file for now.

Reply:
Yes please.. I'm not sure our staging URLs should be all that public or in our config.

Reply:
Done.